tests/unit/styleUniformsTests.cpp

```diff
@@ -0,0 +1,109 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+#include <iostream>
+#include <vector>
+#include <string>
+
+#include "data/filters.h"
+#include "yaml-cpp/yaml.h"
+#include "scene/sceneLoader.h"
+#include "scene/scene.h"
+#include "util/variant.h"
+
+using namespace Tangram;
+using YAML::Node;
+
+Scene scene;
+SceneLoader sceneLoader;
+
+TEST_CASE( "Style Mixing Test: Float uniform value", "[mixing][core][yaml]") {
```

Comment:
Don't know that I'd call these style "mixing" tests -- mixing to be is about the actual `mix` functionality. Shader uniforms are an independent feature.